src/pipecat/transports/services/helpers/daily_rest.py

```diff
@@ -111,6 +124,75 @@ class DailyRoomObject(BaseModel):
     config: DailyRoomProperties
 
 
+class DailyMeetingTokenProperties(BaseModel):
+    # https://docs.daily.co/reference/rest-api/meeting-tokens/create-meeting-token#properties
+
+    room_name: Optional[str] = Field(
+        default=None,
+        description="The name of the room this token is for. If not provided, the token can be used for any room.",
+    )
+
+    eject_at_token_exp: Optional[bool] = Field(
+        default=None,
+        description="If `true`, the user will be ejected from the room when the token expires. Defaults to `false`.",
+    )
+    eject_after_elapsed: Optional[int] = Field(
+        default=None,
+        description="The number of seconds after which the user will be ejected from the room. If not provided, the user will not be ejected based on elapsed time.",
+    )
+
+    nfb: Optional[int] = Field(
+        default=None,
+        description="Not before. This is a unix timestamp (seconds since the epoch.) Users cannot join a meeting in with this token before this time.",
+    )
+
+    exp: Optional[int] = Field(
+        default=60 * 60,
```

Comment:
Let's set this to `None` and just make the comment stronger about setting it.

```suggestion
        default=None,
```